IOS/SwiftSeedProject/SwiftSeedProject/Interfaces/Components/NewsCollection/SSNewsCollectionView.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  SSNewsCollectionView.swift
+//  SwiftSeedProject
+//
+//  Created by Lucas Pelizza on 4/18/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import UIKit
+
+final class SSNewsCollectionView: UICollectionView, UICollectionViewDelegate, UICollectionViewDataSource {
+
+    var items: [SSNewsCollectionCellVM] = []
+    weak var viewModelDelegate: SSNewsCollectionViewModel?
+    var cellIdentifier: String?
```

Comment:
If this class will use only `SSNewsCollectionViewCell` cells, does it needs to store cellIdentifier here? Shouldn't it be using the const defined on `SSNewsCollectionViewCell`?